src/NoteWorkspace.ts

```diff
@@ -365,13 +376,13 @@ export class NoteWorkspace {
             preview: false,
           })
           .then(() => {
-            // if we created a new file, hop to line #3
```

Comment:
I noticed this was hard-coded to accommodate the default template. I've update it to be more generic and place the selection at the end of the document. Let me know if I misunderstood what was going on here.

Reply:
Ah, nice.